From fe3b54357eb09ed4b0e772286714281795a48045 Mon Sep 17 00:00:00 2001
From: De Huo <De.Huo@windriver.com>
Date: Tue, 6 Aug 2019 17:11:55 +0800
Subject: [PATCH] Add WRLinux1019 oval check for rule
 package_telnet-server_removed

Add WRLinux1019 oval check for rule package_telnet-server_removed

Upstream-Status: Pending

Signed-off-by: De Huo <De.Huo@windriver.com>
---
 .../oval/wrlinux1019.xml                           | 26 ++++++++++++++++++++++
 1 file changed, 26 insertions(+)
 create mode 100644 linux_os/guide/services/obsolete/telnet/package_telnet-server_removed/oval/wrlinux1019.xml

diff --git a/linux_os/guide/services/obsolete/telnet/package_telnet-server_removed/oval/wrlinux1019.xml b/linux_os/guide/services/obsolete/telnet/package_telnet-server_removed/oval/wrlinux1019.xml
new file mode 100644
index 0000000..9795e27
--- /dev/null
+++ b/linux_os/guide/services/obsolete/telnet/package_telnet-server_removed/oval/wrlinux1019.xml
@@ -0,0 +1,26 @@
+<def-group>
+ <!-- THIS FILE IS GENERATED by create_package_removed.py.  DO NOT EDIT.  -->
+  <definition class="compliance" id="package_telnet-server_removed"
+  version="1">
+    <metadata>
+      <title>Package telnet-server Removed</title>
+      <affected family="unix">
+        <platform>multi_platform_wrlinux</platform>
+      </affected>
+      <description>The RPM package telnet-server should be removed.</description>
+      <reference source="galford" ref_id="20140915" ref_url="test_attestation"/>
+    </metadata>
+    <criteria>
+      <criterion comment="package telnet-server is removed"
+      test_ref="test_package_telnet-server_removed" />
+    </criteria>
+  </definition>
+  <linux:rpminfo_test check="all" check_existence="none_exist"
+  id="test_package_telnet-server_removed" version="1"
+  comment="package telnet-server is removed">
+    <linux:object object_ref="obj_package_telnet-server_removed" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_object id="obj_package_telnet-server_removed" version="1">
+    <linux:name>telnet-server</linux:name>
+  </linux:rpminfo_object>
+</def-group>
-- 
1.9.1

